tests/installation/test_installer.py

```diff
@@ -1164,7 +1163,7 @@ def test_installer_with_pypi_repository(
 
     expected = fixture("with-pypi-repository")
 
-    assert not DeepDiff(expected, locker.written_data, ignore_order=True)
+    assert expected == locker.written_data
```

Comment:
Can we apply the same change in `test_installer_old.py`?

Reply:
Done in https://github.com/python-poetry/poetry/pull/6169/commits/92ed15f1f5a2a29c07baabedcaed53739efa9084, thanks.